docs/toc.json

```diff
@@ -62,6 +62,9 @@
       "title": "Firewall",
       "type": "compute/firewall"
     }, {
+      "title": "Health Check",
+      "type": "compute/healthCheck"
```

Comment:
The docs script formats the the module name based on the folder name, so it outputs `compute/health-check`. Not sure if want to update the script to camel case it or just change it here, but the link in the navigation doesn't work because of this.